.gitignore

```diff
@@ -6,3 +6,5 @@
 .Rdata
 .httr-oauth
 .DS_Store
+.Renviron
```

Comment:
Reminds me that the ones using R a lot could just add `A4D_DATA_ROOT` to their `.Renviron`.
Can be easily done using `usethis::edit_r_environ()`
Otherwise, they need to select the folder for the root path whenever they restart R.

Reply:
oh man true we are so blind if that is that easy this would have solved our problem without an function! should definitely just use this solution and put it there thank you!